snare/cloner.py

```diff
@@ -127,6 +128,7 @@ def _make_filename(self, url):
             self.visited_urls.append(current_url.human_repr())
             file_name, hash_name = self._make_filename(current_url)
             print('name: ', file_name)
+            self.logger.debug('Cloned file: %s', file_name)
```

Comment:
Only error log exists for cloner, you have to set up logger with another level to have debug messages

Reply:
Done